src/main/java/net/sacredlabyrinth/phaed/simpleclans/utils/ChatUtils.java

```diff
@@ -162,4 +184,57 @@ public static void applyLastColorToFollowingLines(@NotNull List<String> lines) {
             }
         }
     }
+
+    public static String formatPrice(double value) {
+        Locale locale = getLocale();
+        String cacheKey = locale.toLanguageTag();
+        DecimalFormatSymbols symbols = SYMBOLS_CACHE.computeIfAbsent(cacheKey, k ->
+                new DecimalFormatSymbols(locale));
+        DecimalFormat format = DECIMAL_FORMAT_CACHE.computeIfAbsent(cacheKey, k -> {
+            try {
+                String pattern = plugin.getSettingsManager().getString(ECONOMY_DECIMAL_FORMAT_PATTERN);
+                return new DecimalFormat(pattern, symbols);
+            } catch (IllegalArgumentException e) {
+                plugin.getLogger().warning("Invalid decimal-format-pattern.");
+                plugin.getLogger().warning("Using default decimal format pattern: " + DEFAULT_DECIMAL_FORMAT_PATTERN);
+                return new DecimalFormat(DEFAULT_DECIMAL_FORMAT_PATTERN, symbols);
+            }
+        });
+        return format.format(value);
+    }
+
+    private static Locale getLocale() {
```

Comment:
There is actually the [same method](https://github.com/RoinujNosde/SimpleClans/blob/263d129325888fb1303dda549986ab43ca93ed13/src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/SettingsManager.java#L123) (your method is more safer, obviously, just suggesting to write your checks inside of that public method). And do you think, creating 3 maps for caching two symbols(en_US), date and economy symbols is a good idea? 👀